R/model.R

```diff
@@ -287,10 +287,6 @@ enw_inits <- function(data) {
 #' @param profile Logical, defaults to `FALSE`. Should the model be profiled?
 #' For more on profiling see the [`cmdstanr` documentation](https://mc-stan.org/cmdstanr/articles/profiling.html). # nolint
 #'
-#' @param stanc_options A list of options to pass to the `stanc_options` of
```

Comment:
might want to keep the description and just update the default bit? (and re-render the documentation)

Reply:
Done in [bfdf487](https://github.com/epiforecasts/epinowcast/pull/117/commits/bfdf487d585d8887865297961502258b3f0cccc7)